tests/test_user_login_unit.py

```diff
@@ -0,0 +1,32 @@
+"""
+Unit tests for user authentication utility
+"""
+
+def check_password_changed(user, new_pass) -> bool:
+    return user.check_password(new_pass)
```

Comment:
Method should have a doctring under it saying what it does, and have type annotations for both `user` and `new_pass`.